test/observe_test.dart

```diff
@@ -106,6 +108,7 @@ void _observeTests(createModel(x)) {
 
     subs.add(t.changes.listen((n) {}));
     expect(t.hasObservers, true);
+    Observable.dirtyCheck();
```

Comment:
Also it should be these tests actually pass in dartium, they only fail on the vm (without these changes)